Exiled.CustomItems/API/Features/CustomItem.cs

```diff
@@ -203,7 +203,46 @@ public static bool TryGet(Pickup pickup, out CustomItem customItem)
         /// </summary>
         /// <param name="id">The ID of the <see cref="CustomItem"/> to spawn.</param>
         /// <param name="position">The <see cref="Vector3"/> location to spawn the item.</param>
+        /// <param name="pickup">The <see cref="Pickup"/> instance of the <see cref="CustomItem"/>.</param>
         /// <returns>Returns a value indicating whether the <see cref="CustomItem"/> was spawned or not.</returns>
+        public static bool TrySpawn(int id, Vector3 position, out Pickup pickup)
+        {
+            pickup = default;
+
+            if (!TryGet(id, out CustomItem item))
+                return false;
+
+            item.Spawn(position, out pickup);
+
+            return true;
+        }
+
+        /// <summary>
+        /// Tries to spawn a specific <see cref="CustomItem"/> at a specific <see cref="Vector3"/> position.
+        /// </summary>
+        /// <param name="name">The name of the <see cref="CustomItem"/> to spawn.</param>
+        /// <param name="position">The <see cref="Vector3"/> location to spawn the item.</param>
+        /// <param name="pickup">The <see cref="Pickup"/> instance of the <see cref="CustomItem"/>.</param>
+        /// <returns>Returns a value indicating whether the <see cref="CustomItem"/> was spawned or not.</returns>
+        public static bool TrySpawn(string name, Vector3 position, out Pickup pickup)
+        {
+            pickup = default;
+
+            if (!TryGet(name, out CustomItem item))
+                return false;
+
+            item.Spawn(position, out pickup);
+
+            return true;
+        }
+
+        /// <summary>
+        /// Tries to spawn a specific <see cref="CustomItem"/> at a specific <see cref="Vector3"/> position.
+        /// </summary>
+        /// <param name="id">The ID of the <see cref="CustomItem"/> to spawn.</param>
+        /// <param name="position">The <see cref="Vector3"/> location to spawn the item.</param>
+        /// <returns>Returns a value indicating whether the <see cref="CustomItem"/> was spawned or not.</returns>
+        [Obsolete("Use TrySpawn method with out parameter instead.")]
```

Comment:
Generally speaking, the `out parameter` is a bit confusing as it's natively a parameter modifier, so it's better to add the `modifier`.